rskj-core/src/main/java/co/rsk/peg/BridgeSupport.java

```diff
@@ -1317,7 +1303,7 @@ private void migrateFunds(
         Address activeFederationAddress,
         List<UTXO> utxosToUse
     ) throws IOException {
-
+        logRetiringFederationBalance(retiringFederationWallet);
```

Comment:
I believe we can pass the balance directly. It is the only thing needed for the method. Wdyt?